components/treeview/stories/treeview.stories.js

```diff
@@ -44,8 +46,149 @@ export default {
 	},
 };
 
+/**
+ * By default, treeviews span the entire width of their parent container and are used within panels.
+ */
 export const Default = TreeViewGroup.bind({});
 Default.args = {
+	items: [
+		{
+			id: "label1",
+			label: "Label 1",
+			link: "#",
+		},
+		{
+			id: "group1",
+			label: "Group 1",
+			link: "#",
+			isOpen: true,
+			items: [
+				{
+					id: "label2",
+					label: "Label 2",
+					link: "#",
+				},
+				{
+					id: "label3",
+					label: "Label 3",
+					link: "#",
+				},
+			],
+		},
+		{
+			id: "group2",
+			label: "Group 2",
+			link: "#",
+			items: [
+				{
+					id: "label3",
+					label: "Label 3",
+					link: "#",
+				},
+				{
+					id: "group3",
+					label: "Group 3",
+					link: "#",
+					items: [
+						{
+							id: "label4",
+							label: "Label 4",
+							link: "#",
+						},
+						{
+							id: "group4",
+							label: "Group 4 (Empty)",
+							link: "#",
+							items: [],
+						},
+					],
+				},
+			],
+		},
+	],
+};
+
+// ********* DOCS ONLY ********* //
+export const Sizing = SizingGroup.bind({});
+Sizing.args = Default.args;
+Sizing.tags = ["autodocs", "!dev"];
+Sizing.parameters = {
+	chromatic: { disableAllSnapshots: true },
+};
+
+
+/**
+ * A tree view with a selected item.
+ */
+export const Selected = Default.bind({});
+Selected.tags = ["autodocs", "!dev"];
+Selected.args = {
+	items: [
+		{
+			id: "layer1",
+			label: "Layer 1",
+			link: "#",
+		},
+		{
+			id: "layer2",
+			label: "Layer 2",
+			link: "#",
+			isSelected: true,
+		},
+	]
+};
+Selected.parameters = {
+	chromatic: { disableAllSnapshots: true },
+};
+
+/**
+ * A tree view with quiet selection.
+ */
+export const Quiet = Default.bind({});
+Quiet.storyName = "Selected (Quiet)";
+Quiet.tags = ["autodocs", "!dev"];
+Quiet.args = {
+	...Selected.args,
+	isQuiet: true,
+};
+Quiet.parameters = {
+	chromatic: { disableAllSnapshots: true },
+};
+
+/**
+ * Detached tree views are meant to be used outside of a panel. Items in detached tree views have rounded corners.
+ */
```

Comment:
Note! This doesn't have rounded corners, but it also doesn't have rounded corners on the docs site. I would consider fixing it here to be out of scope, there's another card for this: [CSS-871](https://jira.corp.adobe.com/browse/CSS-871)